packages/connect-core/src/utils/intent.ts

```diff
@@ -0,0 +1,136 @@
+import { Address } from '@aragon/connect-types'
+import { utils as ethersUtils, providers as ethersProvider } from 'ethers'
+
+import { addressesEqual, normalizeAddress } from './address'
+import { normalizeApp } from './app'
+import { createDirectTransaction } from './transactions'
+import {
+  decodeKernelSetAppParameters,
+  isKernelAppCodeNamespace,
+  isKernelSetAppIntent,
+} from './kernel'
+import { getForwardingPath, getACLForwardingPath } from './path/index'
+import { AppOrAddress, StepDecoded } from '../types'
+import App from '../entities/App'
+import ForwardingPath from '../entities/ForwardingPath'
+
+export async function organizationIntent(
```

Comment:
The only difference between the `organizationIntent` and the `appIntent` is that the one for the Organization creates a direct transaction if the destination is not an installed app. But as we may probably just remove the method for the Organization it's not relevant anymore.